Testing/Data/DocTest/EMU00103638.nxs_v2.md5

```diff
@@ -0,0 +1 @@
+7a1a43edf3e0172536c9995449e41ba5
```

Comment:
Can I just check we have confirmed that it is okay to include these files in our public repository?

Reply:
Will get this confirmed this afternoon.

Reply:
Will be using a different file so I'll update the tests to reflect this.

Reply:
:+1: Let me know when it's been replaced and I'll remove the old ones from the external data store manually.